content/recipes/mikroorm.md

```diff
@@ -0,0 +1,288 @@
+
+```ts
+@Injectable()
+export class MyService {
+
+  constructor(private readonly repo: AuthorRepository) { }
+
+}
+```
+
+#### Load entities automatically
+
+> info **info**  `autoLoadEntities` option was added in v4.1.0 
+
+Manually adding entities to the entities array of the connection options can be 
+tedious. In addition, referencing entities from the root module breaks application 
+domain boundaries and causes leaking implementation details to other parts of the 
+application. To solve this issue, static glob paths can be used.
+
+Note, however, that glob paths are not supported by webpack, so if you are building 
+your application within a monorepo, you won't be able to use them. To address this 
+issue, an alternative solution is provided. To automatically load entities, set the 
+`autoLoadEntities` property of the configuration object (passed into the `forRoot()` 
+method) to `true`, as shown below: 
+
+```ts
+@Module({
+  imports: [
+    MikroOrmModule.forRoot({
+      ...
+      autoLoadEntities: true,
+    }),
+  ],
+})
+export class AppModule {}
+```
+
+With that option specified, every entity registered through the `forFeature()` 
+method will be automatically added to the entities array of the configuration 
+object.
+
+> info **info**  Note that entities that aren't registered through the `forFeature()` method, but 
+> are only referenced from the entity (via a relationship), won't be included by 
+> way of the `autoLoadEntities` setting.
+
+> info **info**  Using `autoLoadEntities` also has no effect on the MikroORM CLI - for that we 
+> still need CLI config with the full list of entities. On the other hand, we can
+> use globs there, as the CLI won't go thru webpack.
+
+#### Request scoped handlers in queues
+
+> info **info**  `@UseRequestContext()` decorator was added in v4.1.0 
+
+As mentioned in the [docs](https://mikro-orm.io/docs/identity-map), we need a clean state for each request. That is handled automatically thanks to the `RequestContext` helper registered via middleware. 
+
+But middlewares are executed only for regular HTTP request handles, what if we need
+a request scoped method outside of that? One example of that is queue handlers or 
+scheduled tasks. 
+
+We can use the `@UseRequestContext()` decorator. It requires you to first inject the
+`MikroORM` instance to current context, it will be then used to create the context 
+for you. Under the hood, the decorator will register new request context for your 
+method and execute it inside the context. 
+
+```ts
+@Injectable()
+export class MyService {
+
+  constructor(private readonly orm: MikroORM) { }
+
+  @UseRequestContext()
+  async doSomething() {
+    // this will be executed in a separate context
+  }
+
+}
+```
+
+#### Using `AsyncLocalStorage` for request context
+
+By default, `domain` api use used in the `RequestContext` helper. Since `@mikro-orm/core@4.0.3`,
+you can use the new `AsyncLocalStorage` too, if you are on up to date node version:
```

Comment:
```suggestion
By default, the `domain` api is use used in the `RequestContext` helper. Since `@mikro-orm/core@4.0.3`,
you can use the new `AsyncLocalStorage` too, if you are on up to date node version:
```